pkgs/top-level/python-packages.nix

```diff
@@ -284,7 +284,9 @@ in {
 
   fire = callPackage ../development/python-modules/fire { };
 
-  globus-sdk = callPackage ../development/python-modules/globus-sdk { };
```

Comment:
You apparently accidentally deleted this line which causes the CI to fail during evaluation:
https://gist.github.com/GrahamcOfBorg/14241c580774a606e627f780e1809db9

Reply:
Done. Thanks for digging into this, I thought that error came from upstream.